e2e-app/src/app/typeahead/focus/typeahead-focus.pw-spec.ts

```diff
@@ -69,6 +74,7 @@ describe('Typeahead', () => {
     if (process.env.BROWSER !== 'webkit') {
       it(`should select element on tab`, async() => {
         await test.page.focus(SELECTOR_TYPEAHEAD);
+        await waitForFocus(SELECTOR_TYPEAHEAD);
```

Comment:
As far as I could test (and as discussed on Wednesday), I think this extra line does not only make this test more reliable on Chromium and Firefox but also makes it pass on `webkit` (which is currently excluded).

Reply:
Yes, I did a helper function in a previous try. Maybe I should re-integrate it, as if the focus('something') is not reliable, it can leads to flaky tests.